docs/content/querying/segmentmetadataquery.md

```diff
@@ -49,6 +49,9 @@ The format of the result is:
   "aggregators" : {
     "metric1" : { "type" : "longSum", "name" : "metric1", "fieldName" : "metric1" }
   },
+  "queryGranularity" : {
+    "type": "all"
```

Comment:
when would "all" make sense? It truncates all timestamps to `JodaUtils.MIN_INSTANT` so they would not fall within any of the segments being generated – right?